src/style.css

```diff
@@ -1 +1,71 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(../src/fonts/Avenir-Book.ttf), url(../src/fonts/Avenir-Heavy.ttf);
+}
+
+.body {
+  margin: 0;
+  box-sizing: border-box;
+  padding-top: 20px;
+  font-family: Avenir, sans-serif;
+}
+
+.input {
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.input__icon::before {
+  content: '';
+  position: absolute;
+  background-image: url(/src/images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  transform-box: content-box;
+}
+
+.input__icon--big::before {
+  width: 20px;
+  height: 20px;
+  top: 25px;
+  left: 26px;
+}
+
+.input__icon--small::before {
+  width: 12px;
+  height: 12px;
+  top: 15px;
+  left: 13px;
+}
+
+.input:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+  transition: box-shadow 0.3s;
```

Comment:
The `transition` property should be declared within the element's main class, not within the hover state, to ensure the transition effect is applied both when the element is hovered over and when the hover state is removed.